docs/testing.rst

```diff
@@ -265,3 +265,91 @@ does hurt to double check though.
 
 .. _Betamax: https://github.com/sigmavirus24/betamax
 .. _cassettes: https://betamax.readthedocs.org/en/latest/cassettes.html
+
+Asserting using the assert statement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``py.test`` allows you to use the standard python ``assert`` for verifying expectations and values in Python tests. For example, you can write the following:
+
+.. code::
```

Comment:
Use `code-block` directives and specify the language for proper highlighting, e.g.,

```reStructuredText
.. code-block:: python

    def f():
        return 3
```